pymchelper/writers/trip98.py

```diff
@@ -1,8 +1,9 @@
 import time
 import logging
 import os
-
 import numpy as np
+from pymchelper import __version__ as _pmcversion
+from pytrip import __version__ as _ptversion
```

Comment:
Another thing - move pytrip import down to the method which needs pytrip version and access to its API (i.e. line 16)